scripts/utils/llvm-gcov.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+# vim: set syntax=python ts=4 :
+
+# Copyright 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+import subprocess
+import sys
+
+subprocess.run(["llvm-cov", "gcov"] + sys.argv[1:],
+               check=True)
```

Comment:
FYI, this will convert any non-zero exit code from the coverage tool to 1 since you're using the Python exception handler, just in case the calling script needs the original value for anything. (The log will still have the original return code printed in the traceback)

Reply:
Done